src/commands/fng.py

```diff
@@ -0,0 +1,49 @@
+from discord import app_commands, Interaction
+
+
+class FNGGroup(app_commands.Group):
+    def __init__(self):
+        super().__init__(name="fng", description="FNG Group")
+
+    @app_commands.command(name="food", description="Outputs the date of the next Friday Night Games with food")
+    async def food(self, interaction: Interaction):
+        import datetime as dt
+
+        #Manually inputting dates
```

Comment:
```suggestion
        # Manually inputting dates
```

Leave a space after the `#` when writing comments. Please make this change for all other comments